R/pkg/DESCRIPTION

```diff
@@ -13,6 +13,7 @@ Authors@R: c(person("Shivaram", "Venkataraman", role = c("aut", "cre"),
 License: Apache License (== 2.0)
 URL: http://www.apache.org/ http://spark.apache.org/
 BugReports: http://spark.apache.org/contributing.html
+OS_Type: unix
```

Comment:
right, I can revert this before merging. I want to have a branch to test devtools install